node.js

```diff
@@ -76,6 +70,7 @@ module.exports = {
           ERROR,
           { argsIgnorePattern: '^_' },
         ],
+        '@typescript-eslint/prefer-optional-chain': ERROR,
```

Comment:
How about WARN for now?

Reply:
I can do WARN, but that doesn't change the fact that inferring Prettier config will be a breaking change itself.

Reply:
Fair call